homeassistant/components/water_heater/__init__.py

```diff
@@ -53,6 +53,7 @@
 STATE_HIGH_DEMAND = "high_demand"
 STATE_HEAT_PUMP = "heat_pump"
 STATE_GAS = "gas"
+STATE_VACATION = "vacation"
```

Comment:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
I'm not qualified for that - I just wanted my hot water heater's features to work.